internal/controller/postgrescluster/pgmonitor.go

```diff
@@ -326,6 +328,13 @@ func addPGMonitorExporterToInstancePodSpec(
 			MountPath: "/conf",
 		}},
 	}
+
+	// set sidecar container resources
+	if sidecars != nil && sidecars.Exporter != nil &&
+		sidecars.Exporter.Resources != nil {
+		exporterContainer.Resources = *sidecars.Exporter.Resources
```

Comment:
The exporter spec already has a field for resources. I like the idea of having all sidecar resources (possibly other fields we want to set on a container) defined in one place. In that case, I'm thinking this section in the exporter spec would need to be considered for removal/upgrade.
https://github.com/CrunchyData/postgres-operator/blob/c654a820bb6d9f4014a393481a89b2aedcca19b0/pkg/apis/postgres-operator.crunchydata.com/v1beta1/postgrescluster_types.go#L534

Reply:
`exporter` is unique among the sidecars as it can be toggled on/off. With that in mind, I think it makes the most sense to leave the existing method of configuring `resources` in place.